packages/types/lib/flow/http.api.ts

```diff
@@ -0,0 +1,12 @@
+import type { ApiError, Endpoint } from '../api';
+import type { IncomingFlowConfigUpgrade } from '../deploy/incomingFlow';
+
+export type UpgradePreBuiltFlow = Endpoint<{
+    Method: 'POST';
+    Path: '/api/v1/flow/upgrade/pre-built';
```

Comment:
see my comment above. I think `PUT /api/v1/flow/prebuilt` would be a better path.